scripts/xenadmin-build.ps1

```diff
@@ -334,10 +334,8 @@ $bomLessEncoding = New-Object System.Text.UTF8Encoding $false
 
 Write-Host "INFO: Generating XCUpdates.xml"
 
-[string]::Format($xmlFormat, $productFullName, $date, $msi_url, $msi_checksum, $productVersion, $source_url) |`
```

Comment:
This was regressing to that issue we had the BOM character that was corrupting the file.

Reply:
Apologies that came from a conflict on the rebase, picked the wrong file to use